README.md

```diff
@@ -152,4 +152,4 @@ data: formBufferProperty('model', Validations, {
 [MIT License](https://github.com/simplabs/ember-simple-auth/blob/master/LICENSE).
 
 `ember-validated-form-buffer` is not an official part of
-[Ember.js](http://emberjs.com) and is not maintained by the Ember.js Core Team.
\ No newline at end of file
+[Ember.js](http://emberjs.com) and is not maintained by the Ember.js Core Team.
```

Comment:
Sorry about that: Github's editor doesn't seem to like this trailing newline...
Well, I guess it's simpler if you fix the typo yourself ;)